tiledb/core.cc

```diff
@@ -64,6 +65,31 @@ bool config_has_key(tiledb::Config config, std::string key) {
   return true;
 }
 
+void check_arr(py::object py_arr) {
+  ptrdiff_t arr_p = py::cast<ptrdiff_t>(py_arr);
+  ArrowArray* array = (ArrowArray*)arr_p;
+  std::cout << "addr: " << array << std::endl;
```

Comment:
Is this std::cout intentional? I think this is a debug function?

Reply:
yes, will remove